src/profiles.jl

```diff
@@ -516,6 +516,7 @@ end
     set!(commp::CommunityProfile, sample::AbstractString, prop::Symbol, val)
 
 Update or insert a value `val` to the `sample` in the CommunityProfile `commp` using a Symbol `prop`. 
+If you want an error to be thrown if the value does not exist, use [`insert!`](@ref).
```

Comment:
Here, it should be "If you want an error to be thrown if the value already exists"

Reply:
Maybe also "...a value `val` to the metadata of `sample` in..."